From 5aa42a3137c657b5eddd4608343aa1854bd7cd27 Mon Sep 17 00:00:00 2001
From: Zhiquan Li <zhiquan.li@windriver.com>
Date: Mon, 16 Nov 2015 10:23:00 +0800
Subject: [PATCH] snort: 2.9.6.0 -> 2.9.7.5

Upstream-Status:Inappropriate [embedded specific]

fix the below error:
checking for daq flow ID... configure:
configure: error: cannot run test program while cross compiling

---
 configure.in | 34 +++++++++++++++++-----------------
 1 file changed, 17 insertions(+), 17 deletions(-)

diff --git a/configure.in b/configure.in
index e7636ce..2fbd298 100644
--- a/configure.in
+++ b/configure.in
@@ -716,23 +716,23 @@ have_daq_address_space_id="yes"
 #        [DAQ version supports address space ID in header.])
 #fi
 
-AC_MSG_CHECKING([for daq flow ID])
-AC_RUN_IFELSE(
-[AC_LANG_PROGRAM(
-[[
-#include <daq.h>
-]],
-[[
-   DAQ_PktHdr_t hdr;
-   hdr.flow_id = 0;
-]])],
-[have_daq_flow_id="yes"],
-[have_daq_flow_id="no"])
-AC_MSG_RESULT($have_daq_flow_id)
-if test "x$have_daq_flow_id" = "xyes"; then
-    AC_DEFINE([HAVE_DAQ_FLOW_ID],[1],
-        [DAQ version supports flow ID in header.])
-fi
+#AC_MSG_CHECKING([for daq flow ID])
+#AC_RUN_IFELSE(
+#[AC_LANG_PROGRAM(
+#[[
+##include <daq.h>
+#]],
+#[[
+#   DAQ_PktHdr_t hdr;
+#   hdr.flow_id = 0;
+#]])],
+have_daq_flow_id="yes"
+#[have_daq_flow_id="no"])
+#AC_MSG_RESULT($have_daq_flow_id)
+#if test "x$have_daq_flow_id" = "xyes"; then
+#    AC_DEFINE([HAVE_DAQ_FLOW_ID],[1],
+#        [DAQ version supports flow ID in header.])
+#fi
 
 AC_MSG_CHECKING([for DAQ_VERDICT_RETRY])
 AC_RUN_IFELSE(
